packages/neuron-wallet/tests/services/tx/transaction-service.test.ts

```diff
@@ -0,0 +1,243 @@
+import os from 'os'
```

Comment:
How about we have a file naming standard to differentiate between the unit tests and integration tests?
Take integration tests for example, we name the test file as `*.intg.test.ts`. If the file contains unit tests, we name as `*.test.ts`.

Reply:
Let's do this systematically.